r1cs/gadgets/std/src/bits/mod.rs

```diff
@@ -18,6 +18,26 @@ pub trait ToBitsGadget<ConstraintF: Field> {
         &self,
         cs: CS,
     ) -> Result<Vec<Boolean>, SynthesisError>;
+
+    /// Outputs the little-endian bit representation of `Self`
+    fn to_bits_le<CS: ConstraintSystemAbstract<ConstraintF>>(
+        &self,
+        cs: CS,
+    ) -> Result<Vec<Boolean>, SynthesisError> {
+        let mut bits = self.to_bits(cs)?;
+        bits.reverse();
+        Ok(bits)
+    }
+
+    /// Converts `Self` to little-endian bit representation, checking if the bit representation is
+    /// 'valid'. The trivial default implementation may be overridden by implementors for which the
+    /// validity check is meaningful (e.g., field gadgets)
+    fn to_bits_strict_le<CS: ConstraintSystemAbstract<ConstraintF>>(
+        &self,
+        cs: CS,
+     ) -> Result<Vec<Boolean>, SynthesisError> {
+        self.to_bits_le(cs)
```

Comment:
Let's do it like in to_bits_le(), i.e. calling to_bits_strict and then returning the reversed bits